tests/libs/datasets/data_source_test.py

```diff
@@ -47,11 +46,12 @@ class CANScraperForTest(can_scraper_usafacts.CANScraperUSAFactsProvider):
         # observations are not sorted by increasing date.
         input_data = input_data.iloc[::-1]
 
-    data = ccd_helpers.CanScraperLoader(input_data)
+    class CANScraperForTest(can_scraper_usafacts.CANScraperUSAFactsProvider):
+        @staticmethod
+        def _get_covid_county_dataset():
```

Comment:
If this is a classmethod, you could probably pretty easily mock out this using mocks rather than needing to create a test class. 

but this method is fine as well!

Reply:
Ha, I thought the subclass and override was preferable to mocking. Getting this working without breaking the cache was a bit fiddly. classmethod doesn't seem to play with lru_cache as well. Cleaning up all this inheritance/class stuff is an exercise for another day.